lib/ui/viewer/file/zoomable_image.dart

```diff
@@ -111,21 +101,16 @@ class _ZoomableImageState extends State<ZoomableImage>
       content = const EnteLoadingWidget();
     }
 
-    final GestureDragUpdateCallback? verticalDragCallback = _isZooming
-        ? null
-        : (d) => {
-              if (!_isZooming)
-                {
-                  if (d.delta.dy > dragSensitivity)
-                    {
-                      {Navigator.of(context).pop()},
-                    }
-                  else if (d.delta.dy < (dragSensitivity * -1))
-                    {
-                      showDetailsSheet(context, widget.photo),
-                    },
-                },
-            };
+    verticalDragCallback(d) => {
+          if (d.delta.dy > dragSensitivity)
```

Comment:
The expectation is that if the photo is zoomed in, swiping down should NOT close the photo, and swiping up should NOT bring up the details-sheet. Could you confirm that after removing the `!_isZooming` check this behavior is intact?

Reply:
it is having no effect, I have checked it. Even after removing and zoomed and when we swipe down or up, same effect